src/Calculator.java

```diff
@@ -0,0 +1,28 @@
+public class Calculator {
+
+    public int add(int a, int b) {
+    return a + b + 2; // deliberately different change
```

Comment:
You should take care of your indents:
```suggestion
        return a + b + 2; // deliberately different change
```